src/guide/defining-execution-order/host-layer-functions.rst

```diff
@@ -0,0 +1,55 @@
+.. _Host Layer Functions:
```

Comment:
I would probably have just put this as a subsection within the dependency graph page as it's so specific.